<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataWindow</class>
 <widget class="QMainWindow" name="DataWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Data</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_back">
        <property name="maximumSize">
         <size>
          <width>5000</width>
          <height>300</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>&lt; Back</string>
        </property>
        <property name="cssClass" stdset="0">
         <string>btn-inverted</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_reset">
        <property name="maximumSize">
         <size>
          <width>5000</width>
          <height>300</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
        <property name="cssClass" stdset="0">
         <string>btn-inverted</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="selection_data">
        <property name="minimumSize">
         <size>
          <width>105</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>5000</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="maxVisibleItems">
         <number>11</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>782</width>
         <height>401</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="content_container">
          <item>
           <widget class="QLabel" name="label_made_cocktails">
            <property name="maximumSize">
             <size>
              <width>5000</width>
              <height>80</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>22</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Cocktail(s) made: XX</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="cssClass" stdset="0">
             <string>header-underline bold</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
